hoodie-common/src/main/avro/HoodieArchivedMetaEntry.avsc

```diff
@@ -0,0 +1,59 @@
+{
+   "type":"record",
+   "name":"HoodieArchivedMetaEntry",
+   "namespace":"com.uber.hoodie.avro.model",
+   "fields":[
+      {
+         "name":"hoodieCommitMetadata",
+         "type":[
+            "null",
+            "HoodieCommitMetadata"
+         ],
+         "default": "null"
+      },
+      {
+         "name":"hoodieCleanMetadata",
+         "type":[
+            "null",
+            "HoodieCleanMetadata"
+         ],
+         "default": "null"
+      },
+      {
+         "name":"hoodieCompactionMetadata",
+         "type":[
+            "null",
+            "HoodieCompactionMetadata"
+         ],
+         "default": "null"
+      },
+      {
+         "name":"hoodieRollbackMetadata",
+         "type":[
+            "null",
+            "HoodieRollbackMetadata"
+         ],
+         "default": "null"
+      },
+      {
+         "name":"hoodieSavePointMetadata",
+         "type":[
+            "null",
+            "HoodieSavepointMetadata"
+         ],
+         "default": "null"
+      },
+      {
+         "name":"commitTime",
+         "type":["null","string"]
+      },
+      {
+         "name":"actionType",
```

Comment:
dealing with Specific Avro records is tricky for the same reasons. Please file follow on issues to fix this and CLI command for reading with the necessary details , before I can merge this,